js-callback-functions_user-login-callback-1-named-vs-anonymous/js/index.js

```diff
@@ -6,3 +6,15 @@ function handleUserLogin(onSuccess) {
 }
 
 // Call handleUserLogin below!
+
+function showWelcomeMessage(x) {
```

Comment:
Here the parameter is not necessary, this function is simply printing the string on the console, there is no need for parameter.